src/views/projects/Patch.svelte

```diff
@@ -425,6 +450,27 @@
     {/if}
   </div>
   <div class="metadata">
+    <div>
+      <div class="metadata-section-header">Reviews</div>
+      <div class="metadata-section-body">
+        {#each currentRevision.reviews as review}
+          <div class="review">
+            <span
+              class:review-accept={review.verdict === "accept"}
+              class:review-reject={review.verdict === "reject"}>
+              <Icon
+                size="small"
+                name={review.verdict === "accept" ? "checkmark" : "cross"} />
```

Comment:
Sometimes there's no verdict, just a comment. Does this take it into account?

Reply:
Yeah haven't pushed the latest commit yet, I check for `accept` or `reject` verdict and else show the chat icon